src/controllers/update/update.continue.saved.filing.controller.ts

```diff
@@ -29,7 +28,8 @@ export const post = (req: Request, res: Response, _: NextFunction) => {
   }
 
   if (isRemoveJourney(req)){
-    return res.redirect(`${config.REMOVE_SOLD_ALL_LAND_FILTER_PAGE}?${JOURNEY_QUERY_PARAM}=${JourneyType.remove}`);
+    return res.redirect(`${config.REMOVE_SOLD_ALL_LAND_FILTER_PAGE}`);
```

Comment:
Because where we're going is a remove specific page, there's no need to pass in the query parameter.

Reply:
do we need to add start=0, we will have to wipe the session data at the start of the journey similar to the secure update filter page

Reply:
ok, we'll have to work at out what point we wipe the session, on update it's the first screen after the continue question.

Reply:
👍 potentially the same point as when update does that, but coming later...